src/goto-instrument/loop_utils.cpp

```diff
@@ -107,3 +108,41 @@ void get_modifies(
     }
   }
 }
+
+// TODO handle aliasing at all
+void get_uses(
+  const local_may_aliast &local_may_alias,
+  const loopt &loop,
+  usest &uses)
+{
```

Comment:
Also, C++ now has rvalue references, which implies that large data structures can be returned without copy. I'd consider returning directly here.